guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,411 @@ Illustration sources:
+### Emergency Recovery
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/recovery-scenario-emergency.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/recovery-scenario-emergency@2x.png"
+   alt-text = ""
+   width = 248
+   height = 180
+   layout = "float-left-desktop"
+%}
+
+"Emergency recovery scenarios" include situations where a user may need to recover their funds in a hostile or catastrophic situation. Examples include:
+
+- App provider or LSP goes out of business
+- App provider or LSP becomes malicious
+- App provider or LSP has a serious technical malfunction
+
+</div>
+
+While this may seem outside the scope of normal thinking for a product, it is reasonable in this situation due to the collaborative nature of lightning channels. While it should be within the incentive of a business such as the LSP or the app developer to always want to help their customer, that can not always be guaranteed. Therefore, this product equips the user to protect their funds even from a failure of the business creating the product. This demonstrates serious commitment to [design principles]({{'/guide/getting-started/principles/' | relative_url}})!
+
+## Recovering with an automatic cloud backup
 
-However, not all wallets from one application can be recovered in another, depending on what standards they support, as covered in the [wallet interoperability]({{ 'guide/designing-products/interoperability' | relative_url }}) section. Lack of interoperability and vendor dependance is an issue for wallets holding bitcoin on the Lightning network. Lightning network wallets can currently only be restored in the application originally used. It's also worth noting that restoring from a [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) only restores the private keys, but no user data like transaction notes, [contacts]({{ '/guide/daily-spending-wallet/contacts/' | relative_url }}) etc.
+This is by far the simplest way to recover the wallet. The user opts to restore from their cloud backup. The app checks the cloud account for a backup and restores the lightning node on the user's device. This works great for [basic recovery scenarios](#basic-recovery).
+
+<div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/import-options.png"
-   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/import-options@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-menu.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-menu@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-menu@2x.png"
+   layout = "shadow"
+   caption = "Beginning of the restore flow."
```

Comment:
This button is taller than the orange buttons elsewhere.

Reply:
Fixed